lib/miq_automation_engine/service_models/miq_ae_service_vm_or_template.rb

```diff
@@ -41,6 +41,7 @@ class MiqAeServiceVmOrTemplate < MiqAeServiceModelBase
     expose :compliances,           :association => true
     expose :last_compliance,       :association => true
     expose :ems_events,            :association => true
+    expose :evacuate,              :override_return => nil
```

Comment:
@andyvesel 
Why are we overriding the return value? Can't the caller just ignore the returned value.
This gives us problems when we starting implementing these via the REST API.